### DISCUSSION TITLE
ActionIcon color doesn't accept color when using useMantineTheme

### DISCUSSION THREAD

**Discussion by Ruslan-I7** [mantinedev/mantine#3455] [Q&A] [answered]
Hello guys, 

ActionIcon color doesn't accept color when using useMantineTheme but at the same time there is written in the doc

> "ActionIcon color prop accepts only keys of `theme.colors` (including colors index reference, i. e. blue.3)."

Well, if I try to add color using `theme.colors` it doesn't work, if I add color by text it does. So, `theme.colors` from `useMantineTheme` should work or not?

**Comment by Sajarin-M** [comment, marked as answer]
The documentation states that the `color` prop only accepts keys from `theme.colors`
`theme.colors` is defined like:
```js
theme={
  colors : {
    blue: ['#E9EDFC', '#C1CCF6', '#99ABF0', /* ... */],
    /* other colors */
  }
}
```
therefore, in this example, you can pass `blue` or its shade, such as `blue.1`
values that are not defined as a key in `theme.colors` are not accepted.

**Reply by Ruslan-I7** [reply]
I got it, so u cannot use `useMantineTheme` for `ActionIcon` because u pass not a key, u pass a certain color this way. Thanks man
